berkeley-function-call-leaderboard/bfcl/__main__.py

```diff
@@ -52,8 +55,20 @@ def models():
     """
     List available models.
     """
+    available_models = set(HANDLER_MAP.keys())
+
+    # If a custom handler setting exists, add it to the
+    handler_config_path = os.getenv("BFCL_HANDLER_CONFIG")
+    if handler_config_path and os.path.exists(handler_config_path):
+        try:
+            with open(handler_config_path) as f:
+                handler_config = json.load(f)
+                available_models.update(handler_config.keys())
+        except Exception as e:
+            print(f"Warning: Error loading custom handler config: {str(e)}")
+
```

Comment:
Previously, you could simply use dictionary mapping to get handler.
For now,  HandlerLoader will controll.

You can set environment variable for `BFCL_HANDLER_CONFIG`.
This environ var. point to the path of json file.
and if this exists, Loader can find the path of json file.

Reply:
This is example of json files
```json
{
    "my-custom-model": {
        "module_path": "/path/to/my_handlers.py",
        "class_name": "MyCustomHandler"
    }
}
```

and example of setting env var.
```
# you can enter this on command"
export BFCL_HANDLER_CONFIG=/path/to/handler_config.json

python -m bfcl generate --model my-custom-model --test-category all
```

Reply:
At `/path/to/my_handlers.py` you can make custom handlers clas
```python
from bfcl.model_handler.base_handler import BaseHandler
from bfcl.model_handler.model_style import ModelStyle

class MyCustomHandler(BaseHandler):
    def __init__(self, model_name, temperature):
        super().__init__(model_name, temperature)
        self.model_style = ...
        ...
```